test/Feature/QuerySpec.hs

```diff
@@ -130,7 +130,11 @@ spec =
         [json| [{"text_search_vector":"'baz':1 'qux':2"}] |]
 
     it "matches with computed column" $
-      get "/items?always_true=eq.true" `shouldRespondWith`
+      get "/items?always_true=eq.true&order=id.asc" `shouldRespondWith`
+        [json| [{"id":1},{"id":2},{"id":3},{"id":4},{"id":5},{"id":6},{"id":7},{"id":8},{"id":9},{"id":10},{"id":11},{"id":12},{"id":13},{"id":14},{"id":15}] |]
+
+    it "order by computed column" $
+      get "/items?order=always_true.asc,id.asc" `shouldRespondWith`
```

Comment:
I know that this does cover the functionality because this request would return an error prior to your code changes. However it might be nice to emphasize the sorting itself by creating a computed column such as `anti_id` which is defined to be the negative value of the id. Then order by `anti_id` descending and check that the results come out ascending by id.